AudioStream.js

```diff
@@ -16,6 +19,8 @@ module.exports = class AudioStream {
     this.paused = false;
     this.ytdl = null;
     this.mpv = null;
+    this.volume = INITIAL_VOLUME;
+    this.setVolume(this.volume);
```

Comment:
`setVolume` modifies `this.volume`
```suggestion
    this.setVolume(this.volume);
```

Reply:
done